src/ui/theme/components/button.ts

```diff
@@ -9,6 +9,7 @@ const ButtonStyle = {
   variants: {
     solid: variantSolid,
     toggle: variantToggle,
+    headerNav: variantHeaderNav,
```

Comment:
why is this a new variant rather than just a solid button?

Reply:
solid button contains styles that are specifically for the toggle buttons.

Reply:
I don't think this needed to be a new variant necessarily, just a separate component that overrides some of the basic button styles. The way I think about this is: If you need a common component with some specific styling, is it going to be re-used all over the app, or is it just for a well defined area? If it is going to be re-used all over then make it a variant or new component in the theme... otherwise, just make a component called `HeaderButton` that renders the `<Button>` underneath and overrides whatever styles you need. I think theme variants/components should really just be for the most common stuff

Reply:
Ok, that makes sense. I will include this in my next update. Thanks!